### PR TITLE
Add checks for arc status and AMA installation for HCI at scale workbook

### PR DESCRIPTION
**Pull request by sayaghosh58**
## PR Checklist
**Add checks for arc status and AMA installation for HCI at scale workbook to avoid inconsistency between single cluster and at-scale workbooks**
-> Add arg query to filter clusters with arc enabled status.
-> Add arg query to filter clusters with AMA installed.

image

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 1]
```diff
@@ -85,6 +85,49 @@
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources"
           },
+          {
+            "id": "10fd2df8-44a3-4c8f-ba3f-e4cfb94ae130",
+            "version": "KqlParameterItem/1.0",
+            "name": "ArcEnabledClusters",
```
[the remark this answers is not in the document]
Comment (sayaghosh58) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook:91]: Yeah. I have made change to hide it now.

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 2]
```diff
@@ -94,7 +137,7 @@
             "multiSelect": true,
             "quote": "'",
             "delimiter": ",",
-            "query": "Event\r\n| where EventLog =~ \"Microsoft-Windows-SDDC-Management/Operational\"\r\n| extend ClusterArmId = _ResourceId\r\n| where ClusterArmId contains \"/Microsoft.AzureStackHCI/\"\r\n| where EventID in (\"3000\", \"3001\", \"3002\", \"3003\", \"3004\")\r\n| where TimeGenerated {timeRange:query}\r\n| distinct ClusterArmId\r\n| union (datatable (ClusterArmId:string)[\"*\"])",
+            "query": "Event\r\n| where EventLog =~ \"Microsoft-Windows-SDDC-Management/Operational\"\r\n| extend ClusterArmId = _ResourceId\r\n| where ClusterArmId contains \"/Microsoft.AzureStackHCI/\"\r\n| extend clusters = extract(\"/clusters/([^/]+)\", 1, ClusterArmId)\r\n| where clusters in~ ({ArcEnabledClusters})\r\n| where clusters in~ ({AMAInstalledClusters})\r\n| where EventID in (\"3000\", \"3001\", \"3002\", \"3003\", \"3004\")\r\n| where TimeGenerated {timeRange:query}\r\n| distinct ClusterArmId\r\n| union (datatable (ClusterArmId:string)[\"*\"])",
```
[the remark this answers is not in the document]
Comment (sayaghosh58) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook]: Updated the regex to extract the clusterId instead of the cluster name.

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 3]
```diff
@@ -17,7 +17,7 @@
             "type": 4,
             "isRequired": true,
             "value": {
-              "durationMs": 14400000
+              "durationMs": 86400000
```
Comment (myunghojung) [inline Workbooks/Azure Stack HCI/AtScale/Insights.workbook:20]: Is this change needed?

Comment (sayaghosh58) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook:20]: This is just the time picker. This won't affect anything. Made it to last 24hrs instead of last 4hrs.

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 4]
```diff
@@ -85,6 +85,52 @@
...
           },
+          {
+            "id": "10fd2df8-44a3-4c8f-ba3f-e4cfb94ae130",
+            "version": "KqlParameterItem/1.0",
+            "name": "ArcEnabledClusters",
+            "type": 2,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "ExtensibilityResources \r\n| where type =~ 'microsoft.azurestackhci/clusters/arcSettings'\r\n| extend property = properties.aggregateState\r\n| where property == 'Connected' or property == 'PartiallyConnected'\r\n| extend clusters = extract(\"/subscriptions/([^/]+)/resourceGroups/([^/]+)/providers/([^/]+)/clusters/([^/]+)\", 0, id)\r\n| project clusters",
```
[the remark this answers is not in the document]
Comment (sayaghosh58) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook]: Done.

**Review by atasims07 — APPROVED** [review]
